src/cascadia/TerminalApp/App.cpp

```diff
@@ -851,15 +843,18 @@ namespace winrt::TerminalApp::implementation
     {
         uint32_t tabIndexFromControl = 0;
         _tabView.Items().IndexOf(tabViewItem, tabIndexFromControl);
-
-        if (tabIndexFromControl == _GetFocusedTabIndex())
-        {
-            _tabView.SelectedIndex((tabIndexFromControl > 0) ? tabIndexFromControl - 1 : 1);
-        }
+		size_t focusedTabIndex = _GetFocusedTabIndex();
 
         // Removing the tab from the collection will destroy its control and disconnect its connection.
         _tabs.erase(_tabs.begin() + tabIndexFromControl);
         _tabView.Items().RemoveAt(tabIndexFromControl);
+
+		if (tabIndexFromControl == focusedTabIndex)
+		{
+			if (focusedTabIndex >= _tabs.size()) focusedTabIndex = _tabs.size() - 1;

```

Comment:
I think the preferred way in this repo is to use curly braces for if, even when there is only one line and have the curly braces in separate lines